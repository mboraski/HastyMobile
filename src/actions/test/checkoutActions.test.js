// import { setDeliveryNotes } from '../checkoutActions';
//
// describe('checkoutActions', () => {
//     const notes = 'notes';
//     it('creates setDeliveryNotes action', () => {
//         expect(setDeliveryNotes(notes)).toMatchSnapshot();
//     });
// });
