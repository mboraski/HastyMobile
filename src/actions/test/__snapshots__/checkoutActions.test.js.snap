// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checkoutActions creates setDeliveryNotes action 1`] = `
Object {
  "payload": "notes",
  "type": "SET_DELIVERY_NOTES",
}
`;
